pkg/api/query/v1.go

```diff
@@ -486,6 +486,15 @@ func (qapi *QueryAPI) series(r *http.Request) (interface{}, []error, *api.ApiErr
 		return nil, nil, &api.ApiError{Typ: api.ErrorBadData, Err: err}
 	}
 
+	var step time.Duration
+	if t := r.FormValue("step"); t != "" {
```

Comment:
Series never had this, no? 

So why we are adding this? (https://prometheus.io/docs/prometheus/latest/querying/api/#finding-series-by-label-matchers)